src/Copy/BrandCopy.elm

```diff
@@ -13,10 +13,10 @@ brandCopy key =
             CopyText "Hospice Hope Stories"
 
         CallToActionDestination ->
-            CopyText "✨"
+            CopyText "01530222079"
```

Comment:
I think this one should be the survey link

Reply:
I think the way Rose has it - the phone is call to action and the contact form is contact.